### DISCUSSION TITLE
Reaktoro integration with GEMS

### DISCUSSION THREAD

**Discussion by martinyang1976** [reaktoro/reaktoro#416] [Q&A]
Dear Reaktoro experts here,

I have a few beginners questions here. 

1) I can see turotials that Reaktoro can import database from GEMS, the question is can Reaktoro simulate cement hydration? or it is designed for other applications. I believe you have quite good documentation, somehow I could not find this exact info.

2) I found in tutorial that Reaktoro can import CEMDATA.18, with Reaktoro solver, what we could do with the cemdata.18? 

3) I still have trouble to import database from GEMS by following the tutorial. see my error sreenshot. how to integrate gems with Reaktoro? I do not see the details at the thermodynamic backend section.

sorry for these beginner questions. hope does not waste too much of your time.

Best regards
Martin

**Comment by allanleal** [comment]
Hi @martinyang1976 - it seems you are using the `Gems` class from Reaktoro v1, from the shared screenshot via email. Unfortunately, that is not supported in Reaktoro v2. You need to rely on ThermoFun in Reaktoro v2 to use the CEMDATA database. 

This tutorial could be helpful:
https://reaktoro.org/applications/geochemistry/cement-simulation-with-solid-solutions.html

**Reply by martinyang1976** [reply]
Thanks Allan, the tutorial works smoothly with ThermoFun database.

appreciate your help

Martin
